src/styles/partials/components/_images.scss

```diff
@@ -0,0 +1,45 @@
+.dg_image-content {
+  display: block;
+  position: relative;
+  img {
+    display: block;
+    object-fit: fill;
+    width: 100%;
+  }
+  .dg_image {
+    display: block;
+    position: relative;
+  }
+}
+.left {
+  float: left;
+  margin-top: $margin-small;
```

Comment:
margin-top, margin-bottom and max-width are the same for left and right, so we should combine those into one block

```scss
.left,
.right {
  /* shared rules between .left and .right*/
}

.left {
/* only rules specific to .left */
}

.right {
/* only rules specific to .right */
}

Reply:
done